Lombiq.UIKit.Widgets/Recipes/Lombiq.UIKit.Widgets.Content.recipe.json

```diff
@@ -0,0 +1,191 @@
+{
+  "name": "Lombiq.UIKit.Widgets.Content",
+  "displayName": "Lombiq UI Kit widgets - Sample Content - Carousel Widget",
+  "description": "Sample content for testing and demonstration of the Lombiq UI kit Widgets module.",
+  "author": "Lombiq Technologies",
+  "website": "https://github.com/Lombiq/Orchard-UIKit",
+  "version": "1.0",
+  "issetuprecipe": false,
+  "categories": [
+    "training"
+  ],
+  "tags": [
+    "developer",
+    "training",
+    "sample"
+  ],
+  "steps": [
+    {
+      "name": "feature",
+      "enable": [
+        "Lombiq.UIKit.Widgets"
+      ]
+    },
+    {
+      "name": "content",
+      "data": [
+        {
+          "ContentItemId": "4p4am9ww8e69w5ew0tpz1b30xb",
+          "ContentItemVersionId": "4ej1g5kses9y15wy66adn2bk5b",
+          "ContentType": "Page",
+          "DisplayText": "Carousel Widget Example",
+          "Latest": true,
+          "Published": true,
+          "ModifiedUtc": "2024-12-26T21:27:42.6026366Z",
+          "PublishedUtc": "2024-12-26T21:27:42.6607451Z",
+          "CreatedUtc": "2024-12-26T21:25:55.3922238Z",
+          "Owner": "44y7k7zehp4tt1ec442eeps34c",
+          "Author": "admin",
+          "TitlePart": {
+            "Title": "Carousel Widget Example"
+          },
+          "AutoroutePart": {
+            "SetHomepage": false,
+            "Disabled": false,
+            "RouteContainedItems": false,
+            "Absolute": false,
+            "Path": "carousel-widget-example"
+          },
+          "FlowPart": {
+            "Widgets": [
+              {
+                "ContentItemId": "4c4pyaj148qhwzc5d10w40mym6",
+                "ContentItemVersionId": null,
+                "ContentType": "CarouselWidget",
+                "DisplayText": null,
+                "Latest": false,
+                "Published": false,
+                "ModifiedUtc": "2024-12-26T21:27:42.6105074Z",
+                "PublishedUtc": null,
+                "CreatedUtc": null,
+                "Owner": null,
+                "Author": "admin",
+                "CarouselWidgetPart": {},
+                "BagPart": {
+                  "ContentItems": [
+                    {
+                      "ContentItemId": "4kdv73665s30r58kya16gsbzx9",
+                      "ContentItemVersionId": null,
+                      "ContentType": "Slide",
+                      "DisplayText": "About",
+                      "Latest": false,
+                      "Published": false,
+                      "ModifiedUtc": "2024-12-26T21:27:42.6146671Z",
+                      "PublishedUtc": null,
+                      "CreatedUtc": null,
+                      "Owner": "4d1m8j4trsk4px1att6bte204r",
+                      "Author": "admin",
+                      "TitlePart": {
+                        "Text": "About"
+                      },
+                      "SlidePart": {
+                        "Image": {
+                          "Paths": [
+                            "about-bg.jpg"
```

Comment:
These images won't be loaded because these are not part of the recipe so won't be imported. Add a media step where you also upload these images and make them part of the repo as well.